scripts/west_commands/patch.py

```diff
@@ -224,6 +238,10 @@ def apply(self, args, yml):
         patched_mods = set()
 
         for patch_info in patches:
+            mod = patch_info["module"].rstrip('/')
```

Comment:
Would it make sense to create a Path object out of mod instead of calling rstrip?